inigomarquinez/kata-ohce
# Added userLoop tests and functionality

### Type:
* [X] feature: added userLoop functionality to read user input and react accordingly.
* [X] refactor: moved auxiliary methods under lib folder.
* [X] other: userLoop tests done. README updated.

### What's the focus of this PR:
- Add userLoop feature so all the functionality of the ohce CLI is covered.

### How to review this PR:
- Check userLoop tests (stub for console.log and process.exit) and userLoop functionality.
- Check main method calling the new functionality.
